src/anshitsu/retouch.py

```diff
@@ -12,9 +12,10 @@ class Retouch:
 
     Passing an image and options to the constructor will convert the specified image.
     """
-    RGB_RED = 255
-    RGB_GREEN = 255
-    RGB_BLUE = 255
+    RGB_RED_VALUE: int = 255
+    RGB_GREEN_VALUE: int = 255
+    RGB_BLUE_VALUE: int = 255
+    RGB_WHITE_COLOR: Tuple[int] = (RGB_RED_VALUE, RGB_GREEN_VALUE, RGB_BLUE_VALUE)
```

Comment:
The type is `Tuple[int]`, but it should be `Tuple[int,int,int]`.

Reply:
@huideyeren Sorry, thanks!